addons/website_sale/views/website_sale_delivery_templates.xml

```diff
@@ -60,6 +60,12 @@
         <xpath expr="//div[@id='payment_method']" position="attributes">
                 <attribute name="t-att-style">'display: none!important' if not deliveries and delivery_has_storable else ''</attribute>
         </xpath>
+        <xpath expr="//div[@id='payment_method']" position="before">
+            <div t-if="not deliveries and delivery_has_storable and providers" class="alert alert-warning">
```

Comment:
Why `providers`?

Reply:
I think I needed it 4 months ago, but apparently not anymore :) Thx !